src/app/exams/exams-view.scss

```diff
@@ -48,4 +50,40 @@
     outline: none;
     max-width: 50%;
   }
+
+  .rating-scale-keypad {
+    display: flex;
+    justify-content: flex-start;
+    margin: 20px 0;
+  }
+
+  .rating-scale-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 6px;
```

Comment:
This gap feels a little tight to me.

Material says 8px (which is close enough to 6 to not bother)
https://m2.material.io/design/layout/spacing-methods.html#containers-and-ratios

This article which links to a study says 24-36px for that size button
https://uxmovement.com/mobile/optimal-size-and-spacing-for-mobile-buttons/

You mentioned this is targeted for phones so accuracy might be an issue if the buttons are too close together.